# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cloop/makefile-install-noconfvars.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2004-05-08 15:39:45.882594296 +0200
+++ ./Makefile	2004-05-08 16:25:36.674410376 +0200
@@ -6,7 +6,6 @@
 CFLAGS:=-Wall -Wstrict-prototypes -Wno-trigraphs -O2 -s -I. -fno-strict-aliasing -fno-common -fomit-frame-pointer 
 else
 include $(KERNEL_DIR)/.config
-include $(KERNEL_DIR)/conf.vars
 endif
 
 CKERNOPS:=-D__KERNEL__ -DMODULE -fno-builtin -nostdlib -DKBUILD_MODNAME="cloop" -DKBUILD_BASENAME="cloop"
@@ -50,3 +49,9 @@
 	$(CC) -I$(KERNEL_DIR)/include -I$(KERNEL_DIR)/include/asm/mach-default $(CFLAGS) $(CKERNOPS) $< -c -o $@
 
 compressed_loop.o create_compressed_fs.o: compressed_loop.h
+
+install:
+	mkdir -p $(MODDIR)
+	install -m 755 cloop.o $(MODDIR)
+	install -m 755 create_compressed_fs $(PREFIX)/sbin
+	install -m 755 extract_compressed_fs $(PREFIX)/sbin
